mozilla-release/browser/components/sessionstore/SessionStore.jsm

```diff
@@ -488,6 +488,9 @@ var SessionStore = {
 Object.freeze(SessionStore);
 
 const cliqz_removeDuplicatedEntries = function(tabs, overwriteTabs) {
+  if (!Array.isArray(tabs)) {
```

Comment:
what are expected value of tabs? 
`undefined` can be dealt with default value `tabs = []`;

Reply:
@tsl143 We always expect tabs to be an array.
But expectation can not guarantee it will be that.
Let's leave this check instead of default value in args.
Since it is more obvious.

Reply:
Okay, then let's use `return true`

Reply:
I am not sure we can return true here since winData.tabs expects array.
Please take a look down below at the place this function is called.